test/test_snakemake_pipeline.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from __future__ import absolute_import 
```

Comment:
you should also add a test to `test_docker_entrypoint_script` - e.g. see how `germline_variant_cli_args` is handled there and add a case for seq2hla